packages/api/src/controllers/playback.ts

```diff
@@ -49,13 +49,14 @@ const newPlaybackInfo = (
 const getAssetPlaybackUrl = async (
   config: Request["config"],
   ingest: string,
-  id: string
+  id: string,
+  user?: User
```

Comment:
Turns out this argument was never set on the call-site. I think this never worked and IDK how the tests didn't catch it 😱